infrastructure/main.tf

```diff
@@ -18,7 +18,7 @@ data "azurerm_key_vault" "bar_key_vault" {
   resource_group_name = "${local.vault_rg_name}"
 }
 
-data "azurerm_key_vault_secret" "s2s_secret" {
+data "azurerm_key_vault_secret" "S2S-SECRET" {
```

Comment:
There is no reason for this rename , if you want to, change this as well https://github.com/hmcts/bar-apps/pull/280/files#diff-1031eeab417cd8afe77e7e005717d38dR45

Reply:
resolved